<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitialDialog</class>
 <widget class="QDialog" name="InitialDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Valige fail. Uue loomiseks valige uus. Mõne teise avamiseks valige Lehitse...</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="competitionNameLabel">
       <property name="text">
        <string>Võistluse nimi:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="competitionNameEdit">
       <property name="toolTip">
        <string>Praegu valitava võistluse nimi</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="timePlaceLabel">
       <property name="text">
        <string>Aeg ja koht:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="timePlaceEdit">
       <property name="toolTip">
        <string>Toimumise aeg ja koht</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="fileNameLabel">
       <property name="text">
        <string>Faili nimi:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="fileNameEdit">
       <property name="toolTip">
        <string>Võistluse faili nimi</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="browseButton">
       <property name="toolTip">
        <string>Siia vajutades saate otsida teise faili, mida avada</string>
       </property>
       <property name="text">
        <string>Lehitse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>32</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>128</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="newButton">
       <property name="toolTip">
        <string>Siia vajutades saate luua uue võistluse ja faili</string>
       </property>
       <property name="text">
        <string>Uus</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="forwardButton">
       <property name="toolTip">
        <string>Selle võistluse ja faili avamiseks vajutage siia</string>
       </property>
       <property name="text">
        <string>Edasi</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="text">
        <string>Välju</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>forwardButton</sender>
   <signal>clicked()</signal>
   <receiver>InitialDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>InitialDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseButton</sender>
   <signal>clicked()</signal>
   <receiver>InitialDialog</receiver>
   <slot>open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
